File: sdk/approute-sdk/python/approute/enums/funding.py
from enum import Enum


class FundingMethodCode(str, Enum):
    USDT_TRC20 = "USDT_TRC20"
    USDT_BEP20 = "USDT_BEP20"
    USDT_TON = "USDT_TON"
    USDT_BYBIT = "USDT_BYBIT"


class FundingStatus(str, Enum):
    PENDING = "pending"
    CONFIRMING = "confirming"
    SUCCESS = "success"
    FAIL = "fail"
    EXPIRED = "expired"

File: sdk/approute-sdk/golang/enum/funding.go
package enum

// FundingMethodCode identifies a deposit method.
type FundingMethodCode string

const (
	FundingUSDT_TRC20 FundingMethodCode = "USDT_TRC20"
	FundingUSDT_BEP20 FundingMethodCode = "USDT_BEP20"
	FundingUSDT_TON   FundingMethodCode = "USDT_TON"
	FundingUSDT_BYBIT FundingMethodCode = "USDT_BYBIT"
)

// FundingStatus represents the lifecycle status of a funding invoice.
type FundingStatus string

const (
	FundingPending    FundingStatus = "pending"
	FundingConfirming FundingStatus = "confirming"
	FundingSuccess    FundingStatus = "success"
	FundingFail       FundingStatus = "fail"
	FundingExpired    FundingStatus = "expired"
)
